modules/007-registrypackages/images/containerd-fe/werf.inc.yaml

```diff
@@ -23,6 +23,11 @@ git:
     stageDependencies:
       setup:
       - '**/*'
+  - add: /modules/007-{{ $.ModuleName }}/images/{{ $.ImageName }}/patches
+    to: /patches
+    stageDependencies:
+      setup:
```

Comment:
It seems that stageDependencies should be `install`. Because patches are used during `install`.

Reply:
yes you right.